apps/aehttp/src/aehttp_api_validate.erl

```diff
@@ -211,6 +211,11 @@ get_param_value("path", Name, Req) ->
     Value = cowboy_req:binding(to_binding(Name), Req),
     {ok, Value, Req}.
 
+read_body(Req0, Acc) ->
+    case cowboy_req:read_body(Req0) of
+        {ok, Data, Req} -> {ok, <<Acc/binary, Data/binary>>, Req};
+        {more, Data, Req} -> read_body(Req, <<Acc/binary, Data/binary>>)
```

Comment:
Can a malicious client performs a 1GB request to the node?
Can a malicious client stream a neverending request to the node?